pkg/converter/convert_unix.go

```diff
@@ -423,3 +434,259 @@ func Unpack(ctx context.Context, ia content.ReaderAt, dest io.Writer, opt Unpack
 
 	return nil
 }
+
+// LayerConvertFunc returns a function which converts an OCI image layer to a nydus blob layer, and set the media type to "application/vnd.oci.image.layer.nydus.blob.v1".
+func LayerConvertFunc(opt PackOption, backend backend.Backend) converter.ConvertFunc {
```

Comment:
Would be better if put `backend` into `PackOption`.

Reply:
done